src/metrics/fn_args.rs

```diff
@@ -100,3 +124,73 @@ impl NArgs for JavaCode {}
 impl NArgs for GoCode {}
 impl NArgs for CssCode {}
 impl NArgs for HtmlCode {}
+
+#[cfg(test)]
+mod tests {
+    use std::path::PathBuf;
+
+    use super::*;
+
+    #[test]
+    fn test_function_nargs() {
+        check_metrics!(
+            "def f(a, b):
+                 if a:
+                     return a",
+            "foo.py",
+            PythonParser,
+            nargs,
+            [(nargs, 2, usize)],
+            [(nargs_average, 2.0)] // 1 function
+        );
+    }
+
+    #[test]
+    fn test_functions_nargs() {
+        check_metrics!(
+            "def f(a, b):
+                 if a:
+                     return a
+            def f(a, b):
+                 if b:
+                     return b",
+            "foo.py",
+            PythonParser,
+            nargs,
+            [(nargs, 4, usize)],
+            [(nargs_average, 2.0)] // 2 functions
+        );
+
+        check_metrics!(
+            "def f(a):
+                 if a:
+                     return a
+            def f(a, b, c):
+                 if b:
+                     return b",
+            "foo.py",
+            PythonParser,
+            nargs,
+            [(nargs, 4, usize)],
+            [(nargs_average, 2.0)] // 2 functions
```

Comment:
Let's use 2 and 3 arguments instead of 1 and 3, having the same expected values for almost all tests feels risky